src/Microsoft.PowerShell.ConsoleHost/host/msh/StartTranscriptCmdlet.cs

```diff
@@ -228,7 +228,18 @@ protected override void BeginProcessing()
                         System.IO.File.Delete(effectiveFilePath);
                     }
                 }
+                else 
+                {
+                    if(System.IO.Directory.Exists(effectiveFilePath))
```

Comment:
`c:\foo\` might not exist yet -- will this case be taken care of by this change?